content/github/developing-online-with-codespaces/configuring-codespaces-for-your-project.md

```diff
@@ -73,13 +72,17 @@ You can use configuration keys supported by {% data variables.product.prodname_c
 - `remoteEnv`
 - `containerUser`
 - `remoteUser`
-- `updateRemoteUserUID`
 - `mounts`
-- `workspaceMount`
-- `workspaceFolder`
+- `workspaceMount` (Not yet supported)
+- `workspaceFolder` (Not yet supported)
```

Comment:
Because this is a list of keys supported by GitHub Codespaces, folks can assume anything not on the list is unsupported. So instead of adding `(not yet supported)` to these keys, let's remove them entirely. 👍🏻 

```suggestion
```